build.ps1

```diff
@@ -1,10 +1,10 @@
 Param(
     [string] $Configuration = "Debug",
-    [string] $CodeDirectory = ".",
-    [string] $TestsDirectory = ".",
+    [string] $CodeDirectory = "Source",
+    [string] $TestsDirectory = "Source",
     [string] $PackageOutputDirectory = "Build",
     [string] $Version,
-    [string[]] $Package
+    [string[]] $Package = @("Machine.Fakes","Machine.Fakes.FakeItEasy","Machine.Fakes.Moq","Machine.Fakes.NSubstitute","Machine.Fakes.Rhinomocks")

```

Comment:
Perhaps you really want the default action to be to not build the nupkgs, but I see no real benefit - it's pretty quick, so I pulled these defaults from `appveyor.yml`.